File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/SubscribeButton.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class SubscribeButton{

	@SerializedName("subscribeButtonRenderer")
	private SubscribeButtonRenderer subscribeButtonRenderer;

	public void setSubscribeButtonRenderer(SubscribeButtonRenderer subscribeButtonRenderer){
		this.subscribeButtonRenderer = subscribeButtonRenderer;
	}

	public SubscribeButtonRenderer getSubscribeButtonRenderer(){
		return subscribeButtonRenderer;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/SubscribeButton.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import com.google.gson.annotations.SerializedName

class SubscribeButton {
    @SerializedName("subscribeButtonRenderer")
    var subscribeButtonRenderer: SubscribeButtonRenderer? = null
    override fun toString(): String {
        return "SubscribeButton{" +
                "subscribeButtonRenderer = '" + subscribeButtonRenderer + '\'' +
                "}"
    }
}